apps/nextjs-example/.babelrc.js

```diff
@@ -13,6 +12,10 @@ module.exports = {
         unstable_moduleResolution: {
           type: 'commonJS',
           rootDir: path.join(__dirname, '../..'),
+          aliases: {
+            '@/app/*': './app/*',
+            '@/components/*': './components/*',
+          },
```

Comment:
Would this work too?

```suggestion
            '@/*': './*',
          },
```

Reply:
Yes this works.